<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-565S1Q9" failures="1" tests="1" name="Default test" time="71.748" errors="0" timestamp="2024-02-26T16:17:47 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="69.185">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at pageObjects.ProductLandingPage.verifyNumberOfItemsInCart(ProductLandingPage.java:29)
at stepDefinitions.ProductLandingPageSteps.i_validate_that_no_of_items_in_cart_is_increased_by(ProductLandingPageSteps.java:30)
at âœ½.I validate that no of items in cart is increased by 1(file:///E:/Desktop/Assignment/CucumberFramework/src/test/java/features/addProductsToCart.feature:13)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
